components/trades/TradeComposer.tsx

```
@@ -0,0 +1,445 @@
+                                />
+                                {showRecipientDropdown && (displayedKnownContacts.length > 0 || filteredRecipients.length > 0) && (
+                                    <div ref={recipientDropdownRef} className="absolute top-full left-0 mt-1 w-full max-h-60 overflow-y-auto rounded-md bg-zinc-800 shadow-lg z-20 border border-zinc-700">
+                                        {displayedKnownContacts.length > 0 && (
+                                            <div>
+                                                <div className="px-3 py-2 text-xs text-zinc-400 font-semibold">Known Contacts</div>
+                                                {displayedKnownContacts.map(recipient => (
+                                                    <div
+                                                        key={recipient.name}
+                                                        onClick={() => handleSelectRecipient(recipient)}
+                                                        className="flex items-center gap-2 px-3 py-2 hover:bg-zinc-700 cursor-pointer"
+                                                    >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)